README.md

```diff
@@ -47,95 +54,83 @@ page](https://berthub.eu/articles/posts/tracing-app-backend/). Also read:
 Various privacy preserving technologies are being developed, an overview is
 [here](https://berthub.eu/articles/posts/tracing-app-thoughts-and-links/).
 
-What all (or at least most) have in common that infections are reported as a
-series of small keys. Such keys may be reported directly, or via a trusted
-healthcare provider. 
-
-Once a set of keys has been accepted it must be distributed to all users of
-the app. There will likely be millions and millions of app users. There will
-also be hundreds of thousand infected keys in the system.
+How this could work in practice is as follows:
+1. Health care authorities are granted a set of signing keys that can be used for signing (more on this below), in practice this could be implemented by them communicating an activation number to the patient.
+2. Each user's device generates an initial random secret key on the first day and on each subsequent day generates a new secret key with a relation to the previous one (thus creating a chain of secret keys).
```

Comment:
The previous version also included a high-level repeat of it, so I've rewritten it to be actually in-line with the proposals and in a bit higher-level language w/o implementation specifics. However, in retrospect: I do agree that it may be best to leave the definitions in those original document.

I do believe repeating some part of the proposal document is inevitable to define a set of functional requirements for the back-end (otherwise we are missing the language, or are bound to forever jump between documents to find definitions). However, this document may not be the right place for it.